sru-templates/manual/azure-sru

```diff
@@ -123,220 +110,219 @@ az network nsg rule create \
+
+        az network lb rule create \
+            --lb-name dsLB_$UBUNTU_RELEASE  \
+            --name dsLBrule_v4_$UBUNTU_RELEASE  \
+            --resource-group $AZURE_RESOURCE_GROUP  \
+            --frontend-ip-name dsLbFrontEnd_v4_$UBUNTU_RELEASE  \
+            --protocol Tcp  \
+            --frontend-port 22  \
+            --backend-port 22  \
+            --probe-name dsProbe_$UBUNTU_RELEASE \
+            --backend-pool-name dsLbBackEndPool_v4_$UBUNTU_RELEASE
+
+        az network lb rule create \
+            --lb-name dsLB_$UBUNTU_RELEASE  \
+            --name dsLBrule_v6_$UBUNTU_RELEASE  \
+            --resource-group $AZURE_RESOURCE_GROUP  \
+            --frontend-ip-name dsLbFrontEnd_v6_$UBUNTU_RELEASE  \
+            --protocol Tcp  \
+            --frontend-port 22  \
+            --backend-port 22  \
+            --probe-name dsProbe_$UBUNTU_RELEASE \
+            --backend-pool-name dsLbBackEndPool_v6_$UBUNTU_RELEASE
+
+        az vm availability-set create \
+            --name dsAVset_$UBUNTU_RELEASE  \
+            --resource-group $AZURE_RESOURCE_GROUP \
+            --location $AZURE_REGION \
+            --platform-fault-domain-count 2  \
+            --platform-update-domain-count 2
+
+        az network nic create \
+            --name $AZURE_NIC_NAME-$UBUNTU_RELEASE \
+            --resource-group $AZURE_RESOURCE_GROUP \
+            --network-security-group $AZURE_NETWORK_SECURITY_GROUP \
+            --vnet-name $AZURE_VNET_NAME \
+             --accelerated-networking true \
+            --subnet sruSubnet \
+            --private-ip-address-version IPv4 \
+             --lb-address-pools dsLbBackEndPool_v4_$UBUNTU_RELEASE  \
+            --lb-name dsLB_$UBUNTU_RELEASE  \
+            --public-ip-address vmPublicIP-IPv4-$UBUNTU_RELEASE
+
+        az network nic create \
+            --name ${AZURE_NIC_NAME}2-$UBUNTU_RELEASE \
+            --resource-group $AZURE_RESOURCE_GROUP \
+            --network-security-group $AZURE_NETWORK_SECURITY_GROUP \
+            --vnet-name $AZURE_VNET_NAME \
+            --accelerated-networking true \
+            --subnet sruSubnet \
+            --private-ip-address-version IPv4
+
+        # Add IPv6 on primary nic
+        az network nic ip-config create \
+            --name $AZURE_NIC_NAME-$UBUNTU_RELEASE-v6 \
+            --nic-name $AZURE_NIC_NAME-$UBUNTU_RELEASE \
+            --resource-group $AZURE_RESOURCE_GROUP \
+            --vnet-name $AZURE_VNET_NAME \
+            --subnet sruSubnet \
+            --lb-address-pools dsLbBackEndPool_v6_$UBUNTU_RELEASE  \
+            --lb-name dsLB_$UBUNTU_RELEASE \
+            --private-ip-address-version IPv6
+
+    fi
+    az vm create --name=$name \
+        --image=Canonical:UbuntuServer:$image_version:latest \
+        --admin-username=root --admin-username=ubuntu \
+        -g $AZURE_RESOURCE_GROUP --ssh-key-value $SSH_KEY \
+        $AZURE_BOOT_DIAG_ARG \
+        --custom-data ${SAMPLE_CLOUDCONFIG} $nics
```

Comment:
Why the braces on this one and not the others?

Reply:
oops, force of habit from other files. Would like to use the braces everywhere for clarity. But didn't want to undertake this flag day this round.